doc/source/whatsnew/v1.0.0.rst

```diff
@@ -270,7 +270,7 @@ Plotting
 Groupby/resample/rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
--
+- Bug in :meth:`DataFrame.groupby.apply` only returning output from a single group in some cases (:issue:`28652`)
```

Comment:
What are `some cases`? Is it only when returning the index?

Reply:
So far I haven't been able to find another situation that this would fix; should I mention the index case specifically?